cloudflare_worker/worker/src/index.ts

```diff
@@ -243,3 +236,20 @@ async function generateSxgResponse(request: Request, payload: Response) {
   );
   return responseFromWasm(sxg);
 }
+
+async function fetcher(request: WasmRequest): Promise<WasmResponse> {
+  const response = await fetch(
+    request.url,
+    {
+      body: new Uint8Array(request.body),
```

Comment:
Sorry for the naive questions.  Why do you need to do a new Uint8Array here when WasmRequest.body is already a Uint8Array?

Reply:
Thanks for finding this bug. The `WasmRequest.body` should not be a `Uint8Array`, it is now fixed to `number[]`.